scripts/utils.js

```diff
@@ -37,6 +57,29 @@ export const [setLibs, getLibs] = (() => {
  * ------------------------------------------------------------
  */
 
-export async function useMiloSample() {
-  const { createTag } = await import(`${getLibs()}/utils/utils.js`);
+function loadStyle(href, callback) {
+  return loadLink(href, { rel: 'stylesheet', callback });
 }
+
+export async function bootstrapBlock(miloConfigs, blockConfig) {
+  if (!miloConfigs.url) {
+    console.log(`${blockConfig.label} url not found!`);
+    return;
+  }
+  const { miloLibs } = miloConfigs;
+  const { setConfig, createTag } = await import(`${miloLibs}/utils/utils.js`);
+  setConfig({ ...miloConfigs });
+  loadStyle(`${miloLibs}/blocks/${blockConfig.name}/${blockConfig.name}.css`);
+  loadStyle(`${miloLibs}/styles/styles.css`); //TODO: remove when css gets updated in core milo
```

Comment:
We might want to give it a callback to do some logging if either of these fail to load (to make it easier for us to diagnose if consumers have issues). The same probably goes for any other operations that might fail for any reason.

Reply:
Yes, will add some loggers for all failure case.